### DISCUSSION TITLE
How to remove left icons from all right click actions?

### DISCUSSION THREAD

**Discussion by ImGGAAVVIINN** [linuxmint/discussions#491] [Questions]
I dislike the icon on the left when I right-click and prefer to just read the text. I have tried editing all existing .nemo_actions on my machine, but I can't find the .nemo_action corresponding to create new folder and create new files. Is there a way to edit them?
Or I am looking for the wrong thing and there is a way to change it without searching for the .nemo_action?
![actions](https://github.com/linuxmint/discussions/assets/68761233/a4ab1da1-286c-4217-b9c0-88b164617330)

**Comment by mtwebster** [comment]
You don't need to edit anything.  In cinnamon, you can go into Theme settings and turn off menu icons.
